### DISCUSSION TITLE
Start Actions job

### DISCUSSION THREAD

**Discussion by yavgel** [community/community#20116] [Actions]
Could you please implement Actions related features?

**Comment by edward795** [comment]
Hi,currently there are no start & pause buttons for github actions.you can trigger a workflow to run on certain events (commit or push) & manually retrigger it if it fails or manually rerun it if you have a workflow file for your repository.

so all you have to do is **create a workflow file (in yaml)  along with your repo to enable the actions features!**

**Comment by edward795** [comment]
![image](https://user-images.githubusercontent.com/44163764/177686614-05eb5203-032b-43c8-a2a2-a7551d335819.png)
In the **actions** pane, you will have the options once workflow file is updated!
![image](https://user-images.githubusercontent.com/44163764/177686837-c1fc1797-c4dc-4580-87b2-79c26687737b.png)
